lib/ruby_lsp/requests/hover.rb

```diff
@@ -32,16 +32,19 @@ def run
         return unless @document.parsed?
 
         target, _ = locate_node_and_parent(
-          T.must(@document.tree), [SyntaxTree::Command, SyntaxTree::FCall, SyntaxTree::ConstPathRef], @position
+          T.must(@document.tree), [SyntaxTree::Command, SyntaxTree::CallNode, SyntaxTree::ConstPathRef], @position
         )
 
         case target
         when SyntaxTree::Command
           message = target.message
           generate_rails_document_link_hover(message.value, message)
-        when SyntaxTree::FCall
-          message = target.value
-          generate_rails_document_link_hover(message.value, message)
+        when SyntaxTree::CallNode
+          if target.message == :call
+            generate_rails_document_link_hover(target.message.to_s, target)
```

Comment:
Alternatively:
`generate_rails_document_link_hover("call", target)`
🤷‍♂️

Reply:
I think we should ignore `:call` methods instead. I don't believe those will ever be a Rails DSL.

Reply:
I think you're right, I've updated.